ui/src/components/Common/MediaCard/MediaModal/index.tsx

```diff
@@ -0,0 +1,283 @@
+                    {metadata.Rating.map((rating, index) => {
+                      const prefix = rating.image.split('://')[0]
+                      const type = rating.type
+
+                      const iconMap: Record<string, Record<string, string>> = {
+                        imdb: {
+                          audience: '/icons_logos/imdb_icon.svg',
+                        },
+                        rottentomatoes: {
+                          critic: '/icons_logos/rt_critic.svg',
+                          audience: 'icons_logos/rt_audience.svg',
+                        },
+                        themoviedb: {
+                          audience: '/icons_logos/tmdb_icon.svg',
+                        },
+                      }
+
+                      const icon = iconMap[prefix]?.[type]
+                      return (
+                        <div
+                          key={index}
+                          className="mb-1 flex items-center justify-between space-x-1 rounded-lg bg-black bg-opacity-70 px-3 py-1 text-white shadow-lg"
+                        >
+                          <img
+                            src={icon}
+                            alt={`${prefix} ${type} Icon`}
+                            className="h-6 w-6"
+                            title={`${prefix}-${type}`}
+                          />
+                          <span className="flex cursor-default items-center justify-end text-sm font-medium">
+                            {rating.value}
+                          </span>
+                        </div>
+                      )
+                    })}
+                  </div>
+                ) : (
+                  ''
+                )}
+                {metadata.Genre && metadata.Genre.length > 0 ? (
+                  <div className="pointer-events-none absolute bottom-4 right-4 z-10 flex flex-wrap">
+                    {metadata.Genre.map((genre, index) => (
+                      <span
+                        key={index}
+                        className="mr-0.5 flex h-8 items-center justify-center rounded-lg bg-black bg-opacity-70 px-2 font-medium text-white shadow-lg"
+                      >
+                        {genre.tag}
+                      </span>
+                    ))}
+                  </div>
+                ) : (
+                  ''
+                )}
+              </>
+            )}
+          </div>
+
+          <div className="p-4">
+            <div className="flex items-center justify-between border-b pb-4">
+              <div>
+                <h2 className="text-xl font-semibold text-gray-100">
+                  {title} ({year})
+                </h2>
+              </div>
+            </div>
+
+            <div className="mt-2 text-gray-300">
+              <p>{summary || 'No summary available.'}</p>
+            </div>
+
+            <div className="mr-0.5 mt-6 flex flex-row items-center justify-between">
```

Comment:
Adds some margin to stop the button and list items touching eachother:
![image](https://github.com/user-attachments/assets/3ba049ea-a539-410c-b034-225b4b8d2c39)

```suggestion
            <div className="mr-0.5 mt-6 flex flex-row items-center justify-between gap-4">
```

Reply:
Done